content/en/_index.html

```diff
@@ -3,9 +3,7 @@
 description: > 
   Flux is a set of continuous and progressive delivery solutions for Kubernetes, and they are open and extensible. 
   
-  Flux v2 will be GA within the next few months! 
-  
-  This means that Flux v1 will be deprecated before the end of 2021, so now is a good time to start using v2.
+  The APIs of Flux are stable now. This means that now is a good time to start using the new version.
```

Comment:
Rather than saying new version, start migrating from the legacy version.

```markdown
The APIs of Flux are stable now.
Now is the best time to start using Flux, or start your migration if you are a legacy user.
```

Something like that?